src/utils/db/firebase/queries/bookmarkProject.ts

```diff
@@ -0,0 +1,18 @@
+import { CV } from 'models/cv'
+import { compositeProjectId } from 'utils/db/dbUtils'
+import { firestoreAdmin } from 'utils/db/firebase/firebaseAdmin'
+
+const bookmarkProject = (address: string, projectId: number, cv: CV) => {
+  const compositeId = compositeProjectId(projectId, cv)
+  return firestoreAdmin
+    .collection('bookmarks')
```

Comment:
**suggestion (non-blocking)** I guess we'll be using this key elsewhere (for lookups and such) so maybe worth creating a constant for it.

Reply:
it's a function though no? so that's what the compositeProjectId function is supposed to do, unless we want to add it to the project context.